modules/network-observability-flowcollector-view.adoc

```diff
@@ -0,0 +1,88 @@
+
+The following example shows a sample `FlowCollector` resource for {product-title} Network Observability operator:
+[id="network-observability-flowcollector-configuring-about-sample_{context}"]
+.Sample `FlowCollector` resource
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF                                <1>
+    ebpf:
+      sampling: 50                            <2>
+      logLevel: info
+      privileged: false
+      resources:
+        requests:
+          memory: 50Mi
+          cpu: 100m
+        limits:
+          memory: 800Mi
+  processor:
+    logLevel: info
+    resources:
+      requests:
+        memory: 100Mi
+        cpu: 100m
+      limits:
+        memory: 800Mi
+  loki:                                       <3>
+    url: 'http://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network'
+    statusUrl: 'https://loki-query-frontend-http.netobserv.svc:3100/'
+    tenantID: network
+    authToken: HOST
+    tls:
+      enable: true
+      caCert:
+        type: configmap
+        name: loki-ca-bundle
+        certFile: service-ca.crt
+  consolePlugin:
+    register: true
+    logLevel: info
+    portNaming:
+      enable: true
+      portNames:
+        "3100": loki
+    quickFilters:    <4>
+    - name: Applications
+      filter:
+        src_namespace!: 'openshift-,netobserv'
+        dst_namespace!: 'openshift-,netobserv'
+      default: true
+    - name: Infrastructure
+      filter:
+        src_namespace: 'openshift-,netobserv'
+        dst_namespace: 'openshift-,netobserv'
+    - name: Pods network
+      filter:
+        src_kind: 'Pod'
+        dst_kind: 'Pod'
+      default: true
+    - name: Services network
+      filter:
+        dst_kind: 'Service'
+----
+<1> The Agent specification, `spec.agent.type`, must be `EBPF`. eBPF is the only {product-title} supported option.
+<2> You can set the Sampling specification, `spec.agent.ebpf.sampling`, to manage resources. Lower sampling values might consume a large amount of computational, memory and storage resources. You can mitigate this by setting a sampling ratio. A value of 100 means one flow every 100 is sampled. A value of 0 or 1 means all flows are captured. The lower the value, the increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of 50, for example 1:50. Note that more sampled flows also means more storage needed. It is recommend to start with default values and refine empirically, to determine which setting your cluster can manage.
+<3> The Loki specification, `spec.loki` the Loki client. The default values match the Loki install paths mentioned in the Installing the Loki Operator section, but you might have to configure differently if you used another installation method.
```

Comment:
configure it differently?

Reply:
Ah, yea I do think this can be reworked so it seems less vague. However, it will still be vague because we are not providing any installation guidelines if users choose to install Loki from a 3rd party like Helm or Garfana. This documentation only supports installing Loki via the Red Hat Loki Operator